core/Context/ContextInterface.php

```diff
@@ -24,6 +24,8 @@
 
 interface ContextInterface
 {
+    public function hasParameter(string $parameter): bool;
```

Comment:
- [ ] We need to make sure we are not breaking any context implementation

Reply:
I checked this one.
If everyone uses `AbstractContext` - this PR + `toa-core` PR will cover everything.
But I will everything again to be sure